content/sensu-go/5.11/reference/tokens.md

```diff
@@ -36,7 +36,11 @@ Tokens are invoked by wrapping references to entity attributes and labels with d
 - `{{ .name }}` would be replaced with the [entity `name` attribute][3]
 - `{{ .labels.url }}` would be replaced with a custom label called `url`
```

Comment:
@jspaleta seems like lines 37 and 38 should be removed as they are examples of the usage this PR seeks to update. WDYT?

Reply:
From jef: "I want to point out both syntaxes as they will see both in practice when looking at configs in the wild. Index notation is only needed if the key string includes a dot."